lib/tasks/data_hygiene.rake

```diff
@@ -51,42 +51,65 @@ task sync_govdelivery_topic_mappings: :environment do
 
   puts "Fetching topics.."
   topics = Services.gov_delivery.fetch_topics["topics"]
+  topics = topics.map { |topic| [topic["name"], topic["code"]] }
+  subscriber_lists = SubscriberList.where.not(gov_delivery_id: nil)
+                                   .pluck(:title, :gov_delivery_id)
+                                   .map { |title, gov_delivery_id| [(title || gov_delivery_id).strip, gov_delivery_id] }
```

Comment:
For the subscriber lists without titles, GovDelivery gives them names like "MISSING TITLE UKGOVUKDUP_391", so the first time this runs all of these will be considered to have changed. It's a good thing to give them predictable titles ourselves so I think that's fine. We should also investigate at some point why we have lists without titles in the database, but that's a different question...